### PR TITLE
face_circle

### PR DESCRIPTION
**Pull request by robin-oval**
face circle for triangular faces, none otherwise

possible improvement: bestfit circle
not included because there's only a numpy bestfit circle

### CODE REVIEW

File: src/compas/datastructures/mesh/_mesh.py [hunk 1]
```diff
@@ -2633,6 +2633,32 @@ def face_area(self, fkey):
         """
         return area_polygon(self.face_coordinates(fkey))
 
+    def face_circle(self, fkey):
```
Comment (tomvanmele) [inline src/compas/datastructures/mesh/_mesh.py]: not sure if it is advisable to add trimesh specific functions to the general mesh implementation. perhaps we could make this the rule: if N-gon if can be added directly, if trimesh/quadmesh specific => add as separate function...

**Review by tomvanmele — COMMENTED** [review]
not sure if the function should be added to the mesh implementation...

**Comment by robin-oval** [comment]
should I add it into a new file like compas.datastructures.mesh.geometry.py as trimesh_face_circle? along with mesh_face_skewness, mesh_face_aspect_ratio, mesh_vertex_curvature...

**Comment by robin-oval** [comment]
I created a mesh.geometry file and made a trimesh_face_circle function.

**Pull request merged**